modules/migration-adding-replication-repository-to-cam.adoc

```diff
@@ -4,14 +4,18 @@
 // migration/migrating-4-4/migrating-openshift-4_1-to-4.adoc
 // migration/migrating-4-4/migrating-openshift-4_2-to-4.adoc
 [id='migration-adding-replication-repository-to-cam_{context}']
-= Adding a Repository to the CAM web console
+= Adding a replication repository to the CAM web console
 
-You can add an S3 repository to the CAM web console.
+You can add a replication repository to the CAM web console.
```

Comment:
Can you be more descriptive in your intro? This statement only repeats the title.